.editorconfig

```diff
@@ -0,0 +1,18 @@
+# WordPress Coding Standards
+# https://make.wordpress.org/core/handbook/coding-standards/
+	
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
+
+[{.jshintrc,*.json,*.yml}]
```

Comment:
I just checked and there are only 3 JSON files in core:

* https://github.com/xwp/wordpress-develop/blob/master/package.json
* https://github.com/xwp/wordpress-develop/blob/master/src/wp-includes/js/tinymce/skins/lightgray/fonts/tinymce.json
* https://github.com/xwp/wordpress-develop/blob/master/src/wp-includes/js/tinymce/skins/lightgray/fonts/tinymce-small.json

The `package.json` is indented with 2 spaces, whereas the latter two are indented using tabs.

The two `.jshintrc` files in core are both indented with _tabs_: 

* https://github.com/xwp/wordpress-develop/blob/master/.jshintrc
* https://github.com/xwp/wordpress-develop/blob/master/tests/qunit/.jshintrc

So yeah. I think JSON files should be indented with tabs by default, but only `package.json` and a whitelist of other machine-generated JSON files should have 2 space indents.